i18n/zh/metadata.yml

```
@@ -0,0 +1,67 @@
+title: 用于奖励开源生态系统的去中心化协议
+abstract: |-
+ 为所有开源软件创建一个开放、公共和稳定的注册表将使项目能够独立发布版本，而不是
+ 依赖于将这些不规则数据组装到数百个独立（和重复）系统中的第三方。Package维护者将
+ 他们的作品发布到由拜占庭容错区块链驱动的去中心化注册表，以消除单一故障源，提供
+ 稳定的发布，并允许社区来管理其开源生态系统的区域，而不受外部议程的影响。
```

```suggestion
  为所有开源软件创建一个开放、公共和稳定的注册表将使项目能够独立发布版本，而不是
  依赖于将这些不规则数据组装到数百个独立（和重复）系统中的第三方。Package维护者将
  他们的作品发布到由拜占庭容错区块链驱动的去中心化注册表，以消除单一故障源，提供
  稳定的发布，并允许社区来管理其开源生态系统的区域，而不受外部议程的影响。
```

Please properly indent this